roles/display_deployment_plan/templates/plan.j2

```diff
@@ -24,7 +24,12 @@ An OpenShift cluster is about to be deployed. Please double check the provided i
   {{ row_format_str | format('Setup HTTP Store Service', inventory.setup_http_store_service) }}
   {{ row_format_str | format('Setup Assisted Installer', inventory.setup_assisted_installer) }}
   {{ row_format_str | format('Setup PXE Service', inventory.setup_pxe_service) }}
-
+  {# row_format_str | format('Number of PXE Nodes', inventory.num_pxe_nodes if inventory.num_pxe_nodes not '' else inventory.num_pxe_nodes, "[WARNING] No hosts are set." ) #}
+  {%- if inventory.num_pxe_nodes | int > 0 %}
+{{ row_format_str | format('Number of PXE Nodes', inventory.num_pxe_nodes) }}
+  {% else %}
+{{ row_format_str | format('Number of PXE Nodes', messages.pxe_hosts_missing) }}
```

Comment:
Perhaps we want to change the logic here. 
We only need to warn the user there are no pxe nodes and `if setup_pxe_service == true` 
Maybe check for that state and if not just show how many there are.

Reply:
Warnings only show up when setup_pxe_serice is true and num_pxe_nodes is > than 0.